src/test/unit/services/sample/fixed_param_test.cpp

```diff
@@ -153,51 +158,5 @@ TEST_F(ServicesSamplesFixedParam, parameter_checks) {
   EXPECT_DOUBLE_EQ(0.0, parameter_values.back()[2])
     << "final memory_writer should be (0, 0)";
   EXPECT_EQ(return_code, 0);
-
 }
 
-TEST_F(ServicesSamplesFixedParam, output_regression) {
```

Comment:
the invocation of the sampler is the same across all tests.  all of the outputs can be checked in one go.